### PR TITLE
Update build to include PR's in workflows

### PR DESCRIPTION
**Pull request by csmir**
Testing if this runs build

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ on:
     branches-ignore:
       - stable
       - alpha
+  pull_request:
+      branches: [ "alpha" ]
```
Comment (KubaZ2) [inline .github/workflows/build.yml]:
```suggestion
    branches: [ "alpha" ]
```

**Comment by KubaZ2** [comment]
It's not the best idea I think, because on pull request only activates when the pull request is created, it won't trigger for additional pushes to the source branch.

**Review by KubaZ2 — CHANGES_REQUESTED** [review]

**Comment by csmir** [comment]
It is not a bad idea I think, it will rerun the build process every time a new push is made to the pull request. It can even be installed that the PR *must* succeed build before being able to be merged.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -5,6 +5,8 @@ on:
     branches-ignore:
       - stable
       - alpha
+  pull_request:
+    branches: [ "alpha" ]
```
Comment (KubaZ2) [inline .github/workflows/build.yml]:
```suggestion
    branches:
      - stable
      - alpha
```

**Review by KubaZ2 — CHANGES_REQUESTED** [review]

**Comment by KubaZ2** [comment]
> It is not a bad idea I think, it will rerun the build process every time a new push is made to the pull request. It can even be installed that the PR _must_ succeed build before being able to be merged.

You are right, sorry, what do you think about adding `types: [opened, synchronize]`? That's what is used in https://github.com/NetCordDev/NetCord/blob/alpha/.github/workflows/documentation-preview.yml#L5

**Comment by csmir** [comment]
> You are right, sorry, what do you think about adding `types: [opened, synchronize]`? That's what is used in https://github.com/NetCordDev/NetCord/blob/alpha/.github/workflows/documentation-preview.yml#L5

The only thing you lose out on in this circumstance is when the PR is reopened. All further logic remains the same. I honestly do not think it is necessary

**Comment by KubaZ2** [comment]
Not really, there are many activity types:
- assigned
- unassigned
- labeled
- unlabeled
- opened
- edited
- closed
- reopened
- synchronize
- converted_to_draft
- locked
- unlocked
- enqueued
- dequeued
- milestoned
- demilestoned
- ready_for_review
- review_requested
- review_request_removed
- auto_merge_enabled
- auto_merge_disable

Also I think running on reopened is not needed

**Comment by csmir** [comment]
My bad, I read this:
image
and thought that was all it was

**Comment by KubaZ2** [comment]
Well, you are right then, we would only lose reopened there, though I still think reopened is not needed

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -5,6 +5,10 @@ on:
     branches-ignore:
       - stable
       - alpha
+  pull_request:
+    branches:
```
Comment (KubaZ2) [inline .github/workflows/build.yml:10]:
```suggestion
    types: [opened, synchronize]
    branches:
```

**Review by KubaZ2 — CHANGES_REQUESTED** [review]

**Comment by KubaZ2** [comment]
Thanks!

**Pull request merged**